src/components/DeleteConfirmation.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <v-dialog v-model="dialog" persistent max-width="290">
+        <v-card>
+            <v-card-title class="headline">{{lang('delete-confirmation-title')}}</v-card-title>
+            <v-card-text>{{lang('delete-confirmation-content')}}</v-card-text>
+            <v-card-actions>
+                <v-spacer></v-spacer>
+                <v-btn color="green darken-1" flat @click="dialog = false">{{lang('cancel')}}</v-btn>
```

Comment:
Maybe you should change the color of a cancel button to red or vuetify error, because it could be confusing when it has the same color as the delete button